flexinput/src/main/java/com/lytefast/flexinput/utils/BuildUtils.kt

```diff
@@ -0,0 +1,7 @@
+package com.lytefast.flexinput.utils
+
+import android.os.Build
+
+object BuildUtils{
```

Comment:
some nits
- Format this file.
- `isAndroidQ` never changes, just make it a val

Reply:
We can't use a val or the check won't pass lint. Super fucking annoying but here we are on day 5 fixing this bug(s).

Reply:
use `val x : Boolean get() = true` syntax then. It's more kotlin